lib/tasks/scripts/telemedicine_reports.rb

```diff
@@ -0,0 +1,322 @@
+# frozen_string_literal: true
+
+require "csv"
+
+module TelemedicineReports
```

Comment:
This module has two primary methods:
`parse_mixpanel` - reads the mixpanel csv and hydrates it with some facility level info
`generate_report` - uses the mixpanel info, and fetches data from the db to generate the report

All the other methods here are helpers to put the data structures together in the format that the report needs.